person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+    @id = Random.rand(1..1000)
+  end
+
+  def can_use_services?
+    return true if of_age? || parent_permission == true
+
+    false
+  end
+
+  def of_age?
```

Comment:
- [ ] This function should be `is_of_age?` not `of_age?` please try to fix this typo

Reply:
This name throws an error. It's against rubocop linters rules